tests/utilities.py

```diff
@@ -24,34 +24,59 @@
 _ALL_URSULAS = {}
 
 
-def make_ursulas(ether_addresses: list, ursula_starting_port: int, miner_agent=None, miners=False) -> List[Ursula]:
+def make_ursulas(ether_addresses: list, ursula_starting_port: int,
+                 miner_agent=None, miners=False, bare=False) -> Set[Ursula]:
     """
+    :param ether_addresses: Ethereum addresses to create ursulas with.
     :param how_many_ursulas: How many Ursulas to create.
     :param ursula_starting_port: The port of the first created Ursula; subsequent Ursulas will increment the port number by 1.
     :return: A list of created Ursulas
     """
+
     event_loop = asyncio.get_event_loop()
 
     ursulas = set()
     for port, ether_address in enumerate(ether_addresses, start=ursula_starting_port):
-        ursula = Ursula(is_me=True,
-                        ether_address=ether_address,
-                        dht_host="127.0.0.1",
-                        dht_port=port,
-                        db_name="test-{}".format(port),
-                        rest_host="127.0.0.1",
-                        rest_port=port+100,
-                        always_be_learning=False,
-                        miner_agent=miner_agent,
-                        )
-        ursula.attach_rest_server()
-
-        class MockDatastoreThreadPool(object):
-            def callInThread(self, f, *args, **kwargs):
-                return f(*args, **kwargs)
-
-        ursula.datastore_threadpool = MockDatastoreThreadPool()
-        ursula.dht_listen()
+
+        if bare:
```

Comment:
Can you make a comment on what the context of `bare` is here?

Reply:
I think, this is a "script kiddie" (c) Ursula :scream:

Reply:
:heavy_check_mark: